packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the IBM® CICS® Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Documented that exit codes with non-zero values indicate failure.
+- BugFix: Updated deprecated methods for technical currency purposes
```

Comment:
Please add a PR or issue link to any changelog items that will end up in the Zowe Docs release notes.

Reply:
These changes were mostly for V3 conformance purposes, so nothing to add in Zowe Docs 😋